packages/hint/docs/contributor-guide/how-to/formatter.md

```diff
@@ -25,17 +25,19 @@ A `message` looks like this:
 
 ```json
 {
-    "column": "number", // The column number where the issue was found if applicable.
-    "line": "number", // The line number where the issue was found if applicable.
+    "location": {
+        "column": "number", // The column number where the issue was found if applicable.
+        "line": "number", // The line number where the issue was found if applicable.
+    },
     "message": "string", // The human friendly detail of the error.
```

Comment:
@sarvaje There are more properties in Problem type, should we include them here or not necessary for docs?
Actuall type is as follows:
```ts
/** A problem found by a hint  */
export type Problem = {
    /** The location number where the Problem is */
    location: ProblemLocation;
    /** A message providing more information about the Problem */
    message: string;
    /** The html element where the Problem is */
    sourceCode: string;
    /** The uri of the resource firing this event */
    resource: string;
    /** The name of the triggered hint */
    hintId: string;
    /** The category of the triggered hint */
    category: Category;
    /** The severity of the hint based on the actual configuration */
    severity: Severity;
    /** Indicate the language of the sourceCode */
    codeLanguage?: string;
};
```

Reply:
IMO, we should include them.

Reply:
I have added other properties too. And I think it looks good.